website/site/static/admin/config.yml

```diff
@@ -27,6 +27,7 @@ collections: # A list of collections the CMS should be able to edit
           - name: notifications
             label: Notifications
             widget: list
+            allow_add: true
```

Comment:
This won't be necessary with `allow_add` defaulting to true.

Reply:
So this flag should just be removed from config.yml? If so, how would it be editable by users?

Reply:
They would only need to add it if setting the value to `false`.

Reply:
@gazebosx3 This is just our copy of the CMS for editing the website, it doesn't affect what users can do at all.